lib/fragment.js

```diff
@@ -24,12 +31,17 @@ const getAttributes = (tag, context) => {
         Object.assign(attributes, fragmentCtxt);
     }
 
-    return Object.assign({}, attributes, {
-        url: attributes.src,
+    const { src, async, primary, public: isPublic, timeout } = attributes;
```

Comment:
why `public: isPublic`

Reply:
reserved keyword, changing for async as well.